pynoc/cisco.py

```diff
@@ -1,10 +1,32 @@
 """Cisco Switch control object."""
 
 import logging
+import warnings
 
-from six import PY3
 from netaddr import EUI, mac_unix_expanded
-from paramiko import SSHClient, AutoAddPolicy
+from netmiko import ConnectHandler
+
+
+def deprecated(func):
+    """Deprecated decorator.
```

Comment:
Didn't hit this originally on my machine due to outdated `flake8-docstrings`. Fixing now.